### ISSUE TITLE
Adapt to Cthulhu d100 - Providence

### ISSUE THREAD

**Issue by Himmelschrei** [Miskatonic-Investigative-Society/CoC7-FoundryVTT#50]
Hi there!
I wanted to know whether it'd be possible to adapt the system (or fork it) in order to play Cthulhu d100 - Providence. The download from the system is free: https://shadowlands.es/wp-content/uploads/woocommerce_uploads/2020/06/Cthulhu-d100-Carne-quemada-PDF-4-ffdqx5.pdf
It's in spanish, though. If needed, I could help translating. In case there are no plans from your side to adapt it, would you give permission to fork your project and continue from your current work?

Best regards,
Himmelschrei

**Comment by HavlockV** [comment]
Hi Himmelschrei.

I have no knowledge of Cthulhu d100 and therefore no Idea about what it will take to make it compatible. If you can find some english/french doc I can have a look at it. Or at least if you could explain what are the main differences in terms of stats and mechanics. 
Regarding forking the project I have absolutely no objection of any kind, au contraire. By all mean feel free.
Be aware that when I started this my knowledge of the API, Javascript and HTML5/CSS was inexistent. The code has improved a lot as I was learning, but some of the early code is..... not state of the art.

rgds

**Comment by Himmelschrei** [comment]
Hi HavlockV, 

truth be told, I hadn't even heard about the Providence system before I stumbled upon a Spanish publisher house that has a couple of adventures based on it. I don't even know that much of CoC7 myself, as I played just a couple of sessions, so I would be unable to point to you the differences between those systems. There is no English (or French) pdf, as it seems to be just a system used by this publisher house. It doesn't look like there are a lot of differences between those systems (I already know I'm gonna regret having sayed that) at least regarding the character sheet, so I guess I'll just fork your project with your permission and try to adapt it from here. 
My programming skills are quite rusty, though I think you've already done most of the hardwork, so let's see how it goes. 

Thanks a lot.
Regards,
Himmelschrei
